insolar/gen/reference.go

```diff
@@ -28,6 +28,20 @@ func ID() (id insolar.ID) {
 	return
 }
 
+// IDRef generates random id and returns reference on it.
+func IDRef() *insolar.ID {
```

Comment:
it's a handy helper for struct initialization, because, it's not possible to write `id: &gen.ID()`. You shouldn't compare the purity of Go language and test helpers, it's a wrong analogy. In Go you don't have such things, because as a programmer you always can write helper for yourself like I did